src/styles/components/logomark.css

```
@@ -0,0 +1,40 @@
+   Stands in for a real wordmark. Used by the Experience cards and the
+   Companies "worked with" strip. Loaded into @layer components.
+
+   Size is driven by the `--logo-size` custom property: the component sets
+   it inline only when overriding the default (the single dynamic-var
+   exception to the no-inline-styles rule, per AGENTS.md). */
+
+@layer components {
+    .wf {
+        & .logomark {
+            --logo-size: 2.25rem;
+            width: var(--logo-size);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)